### PR TITLE
Add 'Choosing an issue to work on' doc

### PR DESCRIPTION
**Pull request by andybalaam**
---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

### CODE REVIEW

File: docs/choosing-an-issue.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Choosing an issue to work on
+
+So you want to contribute to Element Web? That is awesome!
+
+If you're not sure where to start, make sure you read
+[CONTRIBUTING.md](../CONTRIBUTING.md), and the "Setting up a dev environment"
+section of [README.md](../README.md).
+
+Maybe you've got something specific you'd like to work on? If so, make sure you
+create an issue and discuss it withe the developers before you put a lot of time
```
Comment (SimonBrandner) [inline docs/choosing-an-issue.md:10]:
```suggestion
create an issue and discuss it with the developers before you put a lot of time
```

File: docs/choosing-an-issue.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+into it.
+
+If you're looking for inspiration on where to start, keep reading!
+
+## Finding a good first issue
+
+All the issues for Element Web live in the
+[element-web](https://github.com/vector-im/element-web) repository, including
+issues that actually need fixing in `matrix-react-sdk` or `matrix-js-sdk` or one
```
[the remark this answers is not in the document]
Comment (andybalaam) [reply docs/choosing-an-issue.md]: I removed the reference to matrix-js-sdk since it overcomplicates.

File: docs/choosing-an-issue.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+## Finding a good first issue
+
+All the issues for Element Web live in the
+[element-web](https://github.com/vector-im/element-web) repository, including
+issues that actually need fixing in `matrix-react-sdk` or one of the related
+repos.
+
+The first place to look is for
+[issues tagges with "good first issue"](https://github.com/vector-im/element-web/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
```
Comment (kerryarchibald) [inline docs/choosing-an-issue.md:26]: tagged*

**Review by kerryarchibald — APPROVED** [review]
Spotted one small typo, otherwise looks good! thanks!

**Pull request merged**